### PR TITLE
Turning in something to see what needs more work

### PR DESCRIPTION
**Pull request by Sam-Castaneda13**
Recording 2024-08-20 at 16 26 26

### CODE REVIEW

**Comment by Sam-Castaneda13** [comment]
Recording 2024-08-21 at 09 01 29

File: ts/main.ts [hunk 1]
```diff
@@ -42,25 +57,89 @@ function renderMonster(object: Monster): HTMLDivElement {
     imageUrl = object.name.replace("'", '-');
   }
 
-  const $oneThird = document.createElement('div');
-  $oneThird.className = 'column-one-third';
+  // const $oneThird = document.createElement('div');
+  // $oneThird.className = 'column-one-third';
```
Comment (brandon-moy) [inline ts/main.ts:62]: You should delete this now that we don't need it

File: ts/main.ts [hunk 2]
```diff
@@ -42,25 +57,89 @@ function renderMonster(object: Monster): HTMLDivElement {
...
   const $monName = document.createElement('div');
   $monName.className = 'monster-name';
 
-  const $monsterName = document.createElement('h2');
+  const $monsterName = document.createElement('a');
   $monsterName.textContent = object.name;
+  $monsterName.setAttribute('name', 'plzwork');
+
+  // $oneThird.appendChild($monsterList);
+  $monsterList.appendChild($monImg);
```
Comment (brandon-moy) [inline ts/main.ts:81]: You should delete this now that we don't need it

**Review by brandon-moy — CHANGES_REQUESTED** [review]
Layout is looking a lot better! Got some cleanup work to do before moving on to the next feature though.

**Review by brandon-moy — APPROVED** [review]
✅

**Pull request merged**